src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -378,37 +377,37 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
             {criticality && (
               <Typography>
                 <span className={classes.label}>{t("faultEventMenu.criticality")}:</span>
-                <span className={classes.grey}>{criticality}</span>
+                <span className={classes.notEditable}>{criticality}</span>
```

Comment:
I would go for:
notEditable --> notEditableValue

Reply:
renamed to notEditable